src/kaltura-player.js

```diff
@@ -249,6 +249,13 @@ class KalturaPlayer extends FakeEventTarget {
     return Utils.Object.copyDeep(this._mediaInfo);
   }
 
+  getDrmInfo(): ?ProviderMediaInfoObject {
+    return {
+      keySystem: this.config.drm.keySystem,
```

Comment:
need to take the key system from the adapter, not from config.

Reply:
fixed